packages/shift-next/src/actions/order-actions.js

```diff
@@ -114,7 +114,7 @@ function prepareLineItems (cart) {
     return {
       attributes: {
         sku: lineItem.sku,
-        title: lineItem.title,
+        title: `${lineItem.item.product.title} - ${lineItem.title}`,
```

Comment:
looks good, do we need any composability - Optional Variant title, Product Title, Product +  Variant title?

This would need to pass all the way through the application and be handled as a prop somewhere.

I am guessing there is no pattern of this kind of global config?